dmc_configs/dmc_configs/README.md

```diff
@@ -0,0 +1,176 @@
+# Experiment Configs for Key Figures
+
+Below is a summary of configs that correspond to figures in the Demonstrating Monty Capabilities paper, with descriptions motivating the choice of config parameters.
+
+**Note:** In all experiments, `use_semantic_sensor=False` should be specified. This should be the case once PR #116 is merged, updating the values for PatchViewFinderMountHabitatDatasetArgs etc to be False by default.
+
+## Figure 1 & 2: Diagramatic Figures With No Experiments
+
+## Figure 3: Robust Sensorimotor Inference
+
+Consists of 4 experiments:
+- `dist_agent_1lm` (i.e. no noise)
+- `dist_agent_1lm_noise` - Sensor noise
+- `dist_agent_1lm_randrot` - 14 random rotations  
+- `dist_agent_1lm_randrot_noise`
```

Comment:
I'm sure we talked about this before but just to make sure it is still the best choice: Why are we using the distant agent instead of the surface agent here? I understand why we are using it for the ViT comparison but it seems like here we could use the surface agent.

Reply:
That's a good point, we should get slightly better accuracy with the surface agent. @scottcanoe just thinking do you have any concerns with that? If not I'll add that back.

Reply:
I discussed this with Scott and I think it's worth leaving this for the moment (i.e. using distant agents) so that experiments are consistent across the core figures (3, 4, 6, 7) in terms of what agent we are using. 

That said, we decided we can run the surface agent as well for these configs so that we at least have the results. I was thinking (can discuss more on Monday) we can maybe do a distant-vs-surface comparison in Figure 4 in terms of the impact of model-free (and not just model-based) policy.